README.md

```diff
@@ -186,24 +186,19 @@ wav         - speech files in wave file format
 
 ## Building for Windows on a Linux machine
 
-On Ubuntu 17 and above:
+We recommend using Linux to cross compile for Windows.
+
+On Ubuntu Linux:
    ```
    sudo apt-get install mingw-w64
    mkdir build_windows && cd build_windows
    cmake .. -DCMAKE_TOOLCHAIN_FILE=/home/david/freedv-dev/cmake/Toolchain-Ubuntu-mingw32.cmake -DUNITTEST=FALSE -DGENERATE_CODEBOOK=/home/david/codec2/build_linux/src/generate_codebook 
    make
    ```
    
-## Building for Windows on a Windows machine
-
- ```
- mkdir build_windows (Or what ever you want to call your build dir)
- cmake -G "MinGW Makefiles" -D CMAKE_MAKE_PROGRAM=mingw32-make.exe
- Or if you use ninja for building cmake -G "Ninja" ..
- mingw32-make or ninja  depends on what you used in the last command
- wait for it to build.
- ```
+This will create a working `libcodec2.dll` file for use with other applications (e.g. FreeDV GUI which is in wide spread use on Windows).  Please note the utility/development command line applications (e.g. `freedv_rx.exe`) may not work exactly the same on the Windows CLI compared to running on a Unix machine/shell.  For example pipes may not function as expected, and ctests are not supported.  Our primary development and test environment is Unix, and we lack the resources to support and maintain these applications for other operating systems.
 
+We have anecdotal reports that Windows Subsystem For Linux (WSL) provides better support for the Codec 2 command line applications.
```

Comment:
Probably not necessary to have this here as WSL is effectively a Linux VM (and thus all the normal Codec2 for Linux support applies).